src/ffi/logging.rs

```diff
@@ -17,18 +20,26 @@ use tracing_subscriber::{
 
 use crate::{TelioLogLevel, TelioLoggerCb};
 
+const LOG_BUFFER: usize = 20;
```

Comment:
What should be the value here? Should this be somehow configurable?

Reply:
I guess we can make FeatureConfig value for it with the default to something like 128 or so. What do you think?

Reply:
The problem is that the features are passed in for each [telio/device constructor](https://github.com/NordSecurity/libtelio/blob/main/src/libtelio.udl#L193) while the logger is configured[ completely independently and possibly only once](https://github.com/NordSecurity/libtelio/blob/main/src/libtelio.udl#L157).

Reply:
To me personally, it feels nice to have a big value in here as it feels safer, even though I understand that if we hit even `20` in here it probably means that the log sink is lagging behind and is already problematic.

In theory, once it starts lagging to some degree, probably it will continue the trend and will increase.

I would however feel a bit safer if this value would be slightly higher in case the sink would have a hiccup or two, say value of 1024(purely emotional choice).